recipe-client-addon/content/about-studies/common.js

```diff
@@ -0,0 +1,149 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+"use strict";
+/* eslint-disable no-unused-vars */
```

Comment:
I think technically, this wants to just say it's exporting whatever is being used by the other files? Then local unused vars inside the functions will still get flagged up.